components/renku_data_services/group/api.spec.yaml

```diff
@@ -270,146 +273,134 @@ components:
         id: "01AN4Z79ZS5XN0F25N3DB94T4R"
         name: "R-Project Group"
         slug: "r-project"
-        created_by:
-          id: "owner-KC-id"
-    GroupPost:
+        created_by: "owner-keycloak-id"
+    GroupPostRequest:
       description: Group metadata to be created in Renku
       type: object
       additionalProperties: false
       properties:
         name:
-          $ref: "#/components/schemas/Name"
+          $ref: "#/components/schemas/GroupName"
         slug:
-          $ref: "#/components/schemas/Slug"
+          $ref: "#/components/schemas/GroupSlug"
         description:
-          $ref: "#/components/schemas/Description"
+          $ref: "#/components/schemas/GroupDescription"
       required:
         - "name"
-    GroupPatch:
+        - "slug"
+    GroupPatchRequest:
       type: object
       description: Patch of a group
       additionalProperties: false
       properties:
         name:
-          $ref: "#/components/schemas/Name"
+          $ref: "#/components/schemas/GroupName"
         slug:
-          $ref: "#/components/schemas/Slug"
+          $ref: "#/components/schemas/GroupSlug"
         description:
-          $ref: "#/components/schemas/Description"
+          $ref: "#/components/schemas/GroupDescription"
     Ulid:
       description: ULID identifier
       type: string
       minLength: 26
       maxLength: 26
       pattern: "^[A-Z0-9]{26}$"  # This is case-insensitive
       format: ulid
-    Name:
+    GroupName:
       description: Renku group name
       type: string
       minLength: 1
       maxLength: 99
       example: "My Renku Group :)"
-    Slug:
+    GroupSlug:
       description: A command-line/url friendly name for a group
       type: string
       minLength: 1
       maxLength: 99
       pattern: "^[a-z0-9]+[a-z0-9._-]*$"  # Cannot contain consecutive special characters or end in .git and .atom
       example: "my-renku-group"
-    CreationDate:
-      description: The date and time the group was created (time is always in UTC)
+    GroupCreationDate:
```

Comment:
The same comment for "slug" applies here -- I think we should have only one format for creation dates and use that everywhere, rather than having potentially a different format for each entity.